### PR TITLE
Adding wildfly 10 capability

### PR DESCRIPTION
**Pull request by zamedic**
I our organisation we are starting to use WIldfly 10. 

The xml config files between versions are so different that a standalone.xml from 8 will not work with a standalone.xml from 10.

Here is my thinking of how we can easily manage wildfly config versions and allow the user a simple way of installing wildfly 8 or wildfly 10 with chef. Adding additional config files for each version should be simple. 

basically, I split the templates into the following pattern
#{wildfly['sa']['conf']}.#{version}.erb

I moved the existing template to version 8 and added a standalone-full.xml.10.erb file to get things started.

### CODE REVIEW

**Comment by rbellamy** [comment]
Given that this repo is supporting v10 of Wildfly, is there any reason to keep this PR open? It's an alternative way to allow wrappers to install different versions of Wildfly...
